docs/development/pigs/pig-024.rst

```diff
@@ -0,0 +1,220 @@
+
+- a proper citation of the used Gammapy release,
+- a proper recognition of the achieved work of any contributor,
+- compliance with the FAIR principles for Research Software (`FAIR4RS <https://www.rd-alliance.org/group/fair-research-software-fair4rs-wg/outcomes/fair-principles-research-software-fair4rs>`_).
+
+This PIG aims to set up the project policy about authorship for our citation scheme that is reminded here.
+
+Given the fact that Gammapy is licensed under a 3-clause BSD style license (see gammapy/LICENSE.rst), Gammapy can be used
+and even modified for a science project. For this modified version, the proposed authorship policy of this PIG is not
+applicable but the general citation scheme should be applied.
+
+This PIG is structured as follows: a reminder of our general citation scheme that was up-to-now only given orally and on
+our web pages; then, the authorship policy is given for each of the products associated with Gammapy, namely the intermediate
+releases, the Long Term Support (LTS) releases, the general Gammapy papers, and the contributions to the conference.
+
+2. Reminder of the citation scheme
+==================================
+
+When Gammapy is used for any publication, contribution to conferences or software, authors should cite properly Gammapy.
+It is asked to cite the DOI of the latest Gammapy paper.
+
+.. note::
+    As the Gammapy community fully supports Open Science, we strongly encourage authors using Gammapy to follow
+    the FAIR4RS principles and to allow the reproducibility of their results. As consequence, we suggest always mentioning
+    the `Zenodo <https://zenodo.org/>`_ DOI or the `HAL <https://hal.archives-ouvertes.fr/>`_ identifier (associated with
+    the `Software Heritage <https://www.softwareheritage.org/>`_ archive) of the used release of Gammapy.
+
+This scheme and these two references are precise on our web pages.
+
+3. Authorship policy
+====================
+
+The Gammapy references contain a list of authors, or `contributors`, that requires to be updated with time according to a
+general policy. This section defines what is a contributor to Gammapy and the policy for each type of product.
+
+Definition of a `Contributor`
+-----------------------------
+
+Contributing to Gammapy can be made in different ways:
+
+- contributing to gammapy source code,
+- contributing to our documentation (rst pages, docstring, gammapy-webpage),
+- contributing to our DevOps environment (CI, GitHub configurations, gammapy-docs, release processes, etc),
+- contributing to our associated projects (gammapy-benchmarks, gammapy-data, gammapy-extra, gammapy-cat, gammapy-recipes),
+- contributing to the maintenance of our repositories (conda-forge, PyPI, Zenodo, HAL, docker-hub, etc) or open hub (openhub, etc),
+- leading the precise tasks identified by the Lead Developers,
+- organizing and leading hands-on sessions or schools,
```

Comment:
One has to define what is a contributor in order to make an author list!
A PR is easy to track, whatever the repo. Some DeVOps actions are not associated to a PR, this is more complicated to track.
But, if we have an official list of 'tasks', the associated people will be easily added to the contributor list.

Reply:
I think there is a value in simplicity. Maybe it is simpler to mostly boil down to:
- Official role in the Gammapy project (listed here: https://gammapy.org/team.html)
- Present as author in the main Gammapy git history (without threshold)
- If some contribution is not covered by this, the person could be just required to "opt in"  and submit a PR adding their name to `CITATION.cff`. In which case point two (being part of the git history) holds again.
 
The team / CC / lead devs will then decide to approve the PR justified by the long list of possible contributions you mentioned. But I would add the "opt in" barrier. In 99% of the case it will be uncontroversial to add a person or not...

Reply:
Keeping track of all contributions (beside from official roles and git history) from our side is to much of a burden. So one further argument for the "opt in" system, with team approval (the exact process needs to be decided...)

Reply:
The opt-in option, in addition to the PR and official roles, is very good... and lead to a great simplification of the policy. I like this!

Reply:
I'd also support an opt-in approach for people who want to be on the author list but have only made these "indirect" contributions.

Reply:
This section is about the definition of a contributor.
In the next section, we define what are the authors of the products. And it is mentioned that the list of authors can be extended.

Reply:
Yes, my point is: instead of defining a complex scheme of what a "contributor" is, it might be easier to have a very simple schema with a few things that make you definitely a contributor (actual commits, council, reviews) and then say

> For other forms of contributions, a user can ask the lead developers or the council to be included in the list of contributors